### DISCUSSION TITLE
[Feature] Support negative ratings in EXIF or Sidecar

### DISCUSSION THREAD

**Discussion by chkuendig** [immich-app/immich#14454] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I noticed that Immich ignores negative ratings (-1) in XMP sidecar files. Usually -1 implies rejected (https://exiftool.org/TagNames/XMP.html#xmp ), but Immich seems to force ratings to [0-5 in the asset DTO](https://github.com/immich-app/immich/blob/ba71fd42da5cd948d51fb656bd94dd1336493069/server/src/dtos/asset.dto.ts#L56) and neither imports -1 ratings as rejected. 

![image](https://github.com/user-attachments/assets/47648c7b-8d55-41c5-bc5b-d8c2ce9185d2)

This request is to either (or both):
- allow importing and storing negative ratings to preserve and filter based on this metadata
- automatically flag images with negative ratings as archived/rejected

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Negative ratings have been supported for a while now (#15699). Doing anything automatic with them would fall under #1673.
